### ISSUE TITLE
search result page - when there are no results remove 'unhappy face' from message ... as its not accessible (for screen-readers)

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#116]
![search_result_page_-_when_there_are_no_results_remove_'unhappy_face'_from_message_ _as_its_not_accessibile_(for_screenreaders)](https://user-images.githubusercontent.com/2722522/65528345-18b9f900-deec-11e9-840f-116c2eff0b08.png)
